script.myepisodecalendar/addon.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
 <addon id="script.myepisodecalendar"
        name="MyEpisodeCalendar"
-       version="1.4.0"
+       version="1.4.1"
        provider-name="Connum">
   <requires>
     <import addon="xbmc.python" version="2.14.0"/>
```

Comment:
> WARN: For jarvis it is advised to set xbmc.python version to 2.24.0

Reply:
This is only a warning, it's not a mandatory to change it.

Reply:
Alright, thanks for the info, let me know if there's anyhing you need me to do!